src/main/webapp/issueList/IssueList.js

```diff
@@ -133,25 +184,38 @@ const IssueList = hh(class IssueList extends Component {
     this._isMounted = false;
   }
 
-  init = () => {
+  init = async() => {
     this.props.showSpinner();
     this.tableHandler();
   };
 
-  tableHandler = () => {
-      Project.getProjectByUser(this.paramsContext.get('assignee'), this.paramsContext.get('max')).then(result => {
-        if(this._isMounted) {
-          this.setState(prev => {
-            prev.issues = result.data;
-            return prev;
-          }, () => this.props.hideSpinner())
-        }   
+  tableHandler = async() => {
```

Comment:
nit: here and throughout the file, the indentation and spacing is off. I suggest using the IDE's reformat code feature which will apply the eslint styles.